### PR TITLE
IRichMenuResponse and RichMenuResponse added

### PR DESCRIPTION
**Pull request by sarge963**
**#35 Add RichMenuResponse and IRichMenuResponse** 
Does RichMenuResponse class require Validate() method? I think every object should be "validatable"(despite the fact, this method would not be accessible from outside).

Any advice and suggestions are welcomed.

### CODE REVIEW

File: src/LineBot/RichMenu/RichMenuResponse.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+using System;
+using Newtonsoft.Json;
+
+namespace Line
+{
+    /// <summary>
+    /// Rich menu response object.
+    /// </summary>
+    internal class RichMenuResponse : RichMenu, IRichMenuResponse
```
Comment (dlemstra) [inline src/LineBot/RichMenu/RichMenuResponse.cs]: This class can be sealed.

File: src/LineBot/RichMenu/RichMenuResponse.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    {
+        private string _id;
+
+        [JsonProperty("richMenuId")]
+        public string Id
```
Comment (dlemstra) [inline src/LineBot/RichMenu/RichMenuResponse.cs]: I think we can just do `public string Id { get; set; }` here. This class is only created by a response from the API and that also means we don't need the validation.

File: tests/LineBot.Tests/LineBot.Tests.csproj [hunk 3]
```diff
@@ -21,6 +21,10 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
 
+  <ItemGroup>
```
Comment (dlemstra) [inline tests/LineBot.Tests/LineBot.Tests.csproj]: Can you revert the changes to this file? Don't think we need them.

File: tests/LineBot.Tests/TestHelpers/Tools/StringTools.cs [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+
+namespace Line.Tests
+{
+    [ExcludeFromCodeCoverage]
+    public static class StringTools
+    {
+        public const string Whitespace = " ";
```
Comment (dlemstra) [inline tests/LineBot.Tests/TestHelpers/Tools/StringTools.cs]: I don't think we need this class. It is more clear to have the whitespace in the unit test instead of a variable.

**Comment by sarge963** [comment]
I think, I have sorted out comments mentioned above.
